pkg/controller/controller_utils/controller_utils.go

```diff
@@ -0,0 +1,53 @@
+package controller_utils
```

Comment:
should we just rename this package to controllerName?

Reply:
yeah, I think so.